zhazam/Modules/StartGameTimer/StartGameTimerViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  StartGameTimerViewController.swift
+//  zhazam
+//
+//  Created by Abai Kalikov on 4/22/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+final class StartGameTimerViewController: UIViewController {
+    
+    private enum Constants {
+        static let counterArray: [String] = ["3", "2", "1", ""]
```

Comment:
Don't need to use array here, implement using number(time). function numberOfItems will return number(time). label.text = "\(time-row)"